tests/test_model.py

```diff
@@ -25,6 +26,44 @@ def grey_scale():
     return np.array([[i]*10 for i in range(10)])
 
 
+@pytest.fixture(params=[
+    os.path.join(os.getcwd(), 'resources/data/cvs_avg35_inMNI152.nii.gz'),
+    os.path.join(os.getcwd(), 'resources/data/Lena.png')
+])
+def test_image_path(request):
+    return request.param
+
+
+@pytest.fixture(scope='module', params=[
+    os.path.join(os.getcwd(), 'resources/data/out/saved_image.nii'),
+    os.path.join(os.getcwd(), 'resources/data/out/saved_image.png')
+    ])
+def test_image_write_path(request):
+    if os.path.exists(request.param):
+        os.remove(request.param)
+    return request.param
+
+
+def test_load_image(test_image_path):
+    assert isinstance(mdl.load(test_image_path).shape, tuple)
+
+
+def test_save_image(test_image_path, test_image_write_path):
+    img = mdl.load(test_image_path)
+    mdl.save(img, test_image_write_path)
+    root, ext = os.path.splitext(test_image_write_path)
+    if ext == '.png' and test_image_path.endswith('.nii.gz'):
+        assert os.path.exists(root)
+        assert os.path.exists(os.path.join(root, "saved_image0.png"))
+    else:
+        print("PRINT", test_image_path, test_image_write_path)
```

Comment:
Is it a debug print that is not intended to stay here?

Reply:
yes sure, I remove it